docs/host-and-deploy/index.md

```diff
@@ -55,9 +55,101 @@ Because browsers make requests to Internet-based hosts for client-side pages, we
+In many hosting scenarios, the server's virtual path to the app is the root of the app. In these cases, the app base path is a forward slash (`<base href="/" />`), which is the default configuration for a Blazor app. In other hosting scenarios, such as GitHub Pages and IIS virtual directories, the app base path must be set to the server's virtual path to the app. To set the Blazor app's base path, add or update the `<base>` tag in *index.html* found within the `<head>` tag elements. Set the `href` attribute value to `<virtual-path>/` (the trailing slash is required), where `<virtual-path>/` is the full virtual app root path on the server for the app. In the preceding example, the virutal path is set to `CoolBlazorApp/`: `<base href="CoolBlazorApp/" />`.
+
+For an app with a non-root virtual path configured (for example, `<base href="CoolBlazorApp/" />`), the app fails to find its resources *when run locally*. To overcome this problem during local development and testing, you can supply a *path base* argument that matches the `href` value of the `<base>` tag at runtime.
+
+To pass the path base argument with the root path (`/`) when running the app locally, execute the following command from the Blazor app's directory:
+
+```console
+dotnet run --pathbase=/CoolBlazorApp
+```
+
+The app responds locally at `http://localhost:port/CoolBlazorApp`.
+
+For more information, see the [path base host configuration value](#path-base) section.
+
+> [!IMPORTANT]
+> If a Blazor app uses the [client-side (in-process) hosting model](xref:client-side/blazor/host-and-deploy/hosting-models#client-side-in-process-hosting-model) (based on the **Blazor** project template) and is hosted as an IIS sub-application in an ASP.NET Core app, it's important to disable the inherited ASP.NET Core Module handler. Remove the handler in the Blazor app's published *web.config* file by adding a `<handlers>` section to the file:
+>
+> ```xml
+> <handlers>
+>   <remove name="aspNetCore" />
+> </handlers>
+> ```
+>
+> Removing the handler is performed in addition to configuring the app's base path as described in this section. Set the app base path in the Blazor app's *index.html* file to the IIS alias used when configuring the sub-app in IIS.
+
+## Host configuration values
+
+Blazor apps that use the [server-side hosting model](xref:client-side/blazor/host-and-deploy/hosting-models#server-side-hosting-model) can accept [Web Host configuration values](https://docs.microsoft.com/aspnet/core/fundamentals/host/web-host#host-configuration-values).
+
+Blazor apps that use the [client-side (in-process) hosting model](xref:client-side/blazor/host-and-deploy/hosting-models#client-side-in-process-hosting-model) can accept the following host configuration values as command-line arguments at runtime.
```

Comment:
I believe these are really **dev** host specific settings, right? Once you publish the app it's just a bunch of static files and there is no command-line provided host configuration values involved.